util.go

```diff
@@ -30,3 +30,10 @@ func genSeededGUID2() (uint64, uint64) {
 	n1, n2 := randompool.Pick().TwoInt63()
 	return uint64(n1), uint64(n2)
 }
+
+func IsMetaSpan(s *spanImpl) bool {
+	if _, ok := s.raw.Tags["lightstep.meta_event"]; ok {
```

Comment:
Maybe take advantage of Go's default value semantics

```
return s.raw.Tags["lightstep.meta_event"] != nil
```

Reply:
👍 good call, thanks!